### PR TITLE
Configurable gRPC message size limits

### PR DESCRIPTION
**Pull request by alex-namely**
`MaxCallRecvMsgSize` is the client-side response receive limit.
`MaxCallSendMsgSize` is the client-side request send limit in bytes.

### CODE REVIEW

File: gwy/templates/main.go.tmpl [hunk 1]
```diff
@@ -254,6 +258,18 @@ func SetupMux(ctx context.Context, cfg proxyConfig) *http.ServeMux {
 	logrus.Infof("Proxying requests to gRPC service at '%s'", cfg.backend)
 
 	opts := []grpc.DialOption{grpc.WithInsecure()}
+	
+	if cfg.maxCallRecvMsgSize > 0 || cfg.maxCallSendMsgSize > 0 {
+		callOptions := []grpc.CallOption{}
+		if cfg.maxCallRecvMsgSize > 0 {
```
Comment (vtopc) [inline gwy/templates/main.go.tmpl:264]: Why we are checking this twice?
Could you remove `if cfg.maxCallRecvMsgSize > 0 || cfg.maxCallSendMsgSize > 0 {` check and ...

File: gwy/templates/main.go.tmpl [hunk 2]
```diff
@@ -254,6 +258,18 @@ func SetupMux(ctx context.Context, cfg proxyConfig) *http.ServeMux {
...
+			callOptions = append(callOptions, grpc.MaxCallRecvMsgSize(cfg.maxCallRecvMsgSize))
+		}
+		if cfg.maxCallSendMsgSize > 0 {
+			callOptions = append(callOptions, grpc.MaxCallSendMsgSize(cfg.maxCallSendMsgSize))
+		}
+		opts = append(opts, grpc.WithDefaultCallOptions(callOptions...))
```
Comment (vtopc) [inline gwy/templates/main.go.tmpl:270]: ... and do a check here?
```suggestion
		if len(callOptions) > 0 {
			opts = append(opts, grpc.WithDefaultCallOptions(callOptions...))
		}
```

Comment (alex-namely) [reply gwy/templates/main.go.tmpl:270]: Done

**Review by ido-namely — COMMENTED** [review]
Looks good, but can you document these options in the readme, under the gateway section please?

**Review by abe545 — APPROVED** [review]

**Review by ido-namely — COMMENTED** [review]
sorry, I forgot to comment before - can you update the build version number too?
perhaps after @vtopc's fix?

**Review by ido-namely — APPROVED** [review]

**Review by vtopc — APPROVED** [review]

**Review by vtopc — CHANGES_REQUESTED** [review]
config

**Review by vtopc — APPROVED** [review]

**Pull request merged**